README.md

```diff
@@ -0,0 +1,81 @@
+# Neighbourhood Collaboration Site
+
+We are building a neighbourhood collaboration site and we want to make a system to keep track of people, houses, and addresses of those houses.
+
+## SQL vs NoSQL
+
+Our data will be structured with a predefined schema, therefore we will use an SQL database. SQL databases are relational which will allows us to identify and access data in relation to another piece of data in the database.
+
+## Schema
+
+Table 1 - Owners:
+
+| Owner Name  | Age |
+| ----------- | --- |
+| John Smith  | 45  |
```

Comment:
Good representation visually but I think the schema needs an id and indication of what the primary keys are - would be good to see the relationship of how the different tables in the db are related

Reply:
Agreed, we will add this :)